### PR TITLE
Dev: clean up PluginsController to leave only public action

### PR DESCRIPTION
**Pull request by Shnoulle**

### CODE REVIEW

File: application/controllers/PluginsController.php [hunk 1]
```diff
@@ -7,143 +7,6 @@
...
-    {
-        return $this->properties[$property];
-    }
-
-    public function __set($property, $value)
-    {
-        $this->properties[$property] = $value;
-    }
-
-    public function _init()
```
Comment (Shnoulle) [inline application/controllers/PluginsController.php]: Personnaly : i use direct for own page : but i register BS myself …

File: application/controllers/PluginsController.php [hunk 2]
```diff
@@ -7,143 +7,6 @@
...
-        Yii::app()->bootstrap->init(); // Make sure bootstrap css is rendered in time
-    }
-
-    /**
-     * Activates plugin with $id
-     *
-     * @param int $id
-     * @return void
-     */
-    public function actionActivate($id)
```
Comment (Shnoulle) [inline application/controllers/PluginsController.php]: Ore leave it and redirect to admin controller ? I never use it in plugins (and it's not in API)

File: application/controllers/PluginsController.php [hunk 3]
```diff
@@ -7,143 +7,6 @@
...
-    }
-
-    /**
-     * Show configuration for plugin with $id
-     *
-     * @param int $id
-     * @return void
-     */
-    public function actionConfigure($id)
-    {
```
Comment (Shnoulle) [inline application/controllers/PluginsController.php]: Or leave it and redirect to controller : but it's not in API (i never use it in plugin)

File: application/controllers/PluginsController.php [hunk 4]
```diff
@@ -7,143 +7,6 @@
...
-    }
-
-    /**
-     * Deactivates plugin with $id
-     *
-     * @param int $id
-     * @return void
-     */
-    public function actionDeactivate($id)
-    {
```
Comment (Shnoulle) [inline application/controllers/PluginsController.php]: See other comment about function ;)

File: application/controllers/PluginsController.php (hunk 2 above)
Comment (olleharstedt) [reply application/controllers/PluginsController.php]: I say remove it. It has no place here.

File: application/controllers/PluginsController.php (hunk 3 above)
Comment (olleharstedt) [reply application/controllers/PluginsController.php]: Same.

File: application/controllers/PluginsController.php (hunk 4 above)
Comment (olleharstedt) [reply application/controllers/PluginsController.php]: Same.

**Comment by olleharstedt** [comment]
Can merge this tomorrow.

**Comment by Shnoulle** [comment]
@TonisOrmisson : you can chek it ? @olleharstedt : if you put an OK : i can merge it :)

**Pull request merged**